src/java/com/rapleaf/jack/queries/WhereConstraint.java

```diff
@@ -54,6 +54,6 @@ public Logic getLogic() {
 
   @Override
   public String getSqlStatement() {
-    return (logic != null ? logic.toString() + " " : "") + "`" + column.getSqlKeyword() + "` " + operator.getSqlStatement();
```

Comment:
The backticks are necessary to prevent keyword collisions (PR #138). E.g. if there is a column named `group`, the where clause for this column needs to wrap `group` in these backticks to prevent `MySQLSyntaxErrorException`.